src/build/intro-currency-runtime.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
It comes with a price of fancier build tools. I'm open for PRs implementing this change.